src/Controller/User/InboxController.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mitra\Controller\User;
+
+use Mitra\Dto\DataToDtoManager;
+use Mitra\Dto\Response\ActivityStreams\Activity\CreateDto;
+use Mitra\Dto\Response\ActivityStreams\ArticleDto;
+use Mitra\Dto\Response\ActivityStreams\AudioDto;
+use Mitra\Dto\Response\ActivityStreams\DocumentDto;
+use Mitra\Dto\Response\ActivityStreams\EventDto;
+use Mitra\Dto\Response\ActivityStreams\ImageDto;
+use Mitra\Dto\Response\ActivityStreams\LinkDto;
+use Mitra\Dto\Response\ActivityStreams\MentionDto;
+use Mitra\Dto\Response\ActivityStreams\NoteDto;
+use Mitra\Dto\Response\ActivityStreams\ObjectDto;
+use Mitra\Dto\Response\ActivityStreams\OrderedCollectionDto;
+use Mitra\Dto\Response\ActivityStreams\OrderedCollectionPageDto;
+use Mitra\Dto\Response\ActivityStreams\PlaceDto;
+use Mitra\Dto\Response\ActivityStreams\ProfileDto;
+use Mitra\Dto\Response\ActivityStreams\RelationshipDto;
+use Mitra\Dto\Response\ActivityStreams\TombstoneDto;
+use Mitra\Dto\Response\ActivityStreams\TypeInterface;
+use Mitra\Dto\Response\ActivityStreams\VideoDto;
+use Mitra\Entity\ActivityStreamContentAssignment;
+use Mitra\Entity\User;
+use Mitra\Http\Message\ResponseFactoryInterface;
+use Mitra\Repository\ActivityStreamContentAssignmentRepository;
+use Mitra\Repository\UserRepository;
+use Mitra\Serialization\Encode\EncoderInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Slim\Interfaces\RouteCollectorInterface;
+use Webmozart\Assert\Assert;
+
+final class InboxController
```

Comment:
I did the api call of the inbox via the Postman and asked myself whether the request without a page should not also response show the content?

Reply:
I implemented it that way first but imagine you have 10k content items in your inbox this would slow down the endpoint dramatically and cause very high low and I couldn't imagine a use-case where you want to show every single content item that has ever been in your inbox at once...

Reply:
In which case would the OrderedCollection be needed?

Reply:
The API should be self-explaining this means you should get from endpoint to endpoint with links. The `OrderedCollection` serialized if you call the inbox endpoint without the `page` query param is an entry point for the client. There it will find information on how to access the first page of the inbox (`first` attribute). I agree having an `OrderedCollection` here doesn't make a lot of sense but the inbox endpoint MUST always return either an `OrderedCollection` or an `OrderedCollectionPage`.